### DISCUSSION TITLE
Gitlab owner

### DISCUSSION THREAD

**Discussion by jbadeau** [jreleaser/jreleaser#1060] [Q&A]
I am not able to release to GitLab as the tools check the user projects like this:

```
GET /users/:user_id/projects
```
which returns an empty list as I have no personals repos.

However the GitLab docs say this about the call:

Get a list of visible projects owned by the given user. When accessed without authentication, only public projects are returned.

Only the projects in the user’s (specified in user_id) namespace are returned. Projects owned by the user in any group or subgroups are not returned. An empty list is returned if a profile is set to private.
This endpoint supports [keyset pagination](https://docs.gitlab.com/ee/api/index.html#keyset-based-pagination) for selected order_by options.

I am role "owner" via a group and can release in the GUI but like this it will never work.

Why is releasing limited to the owner who has the repo in its namespace?

**Comment by aalmiray** [comment]
It was likely that we followed the Github/Gitea flow whereas Gitlab has other means (roles) to reach projects. Looks like we'll have to update our GitLab support with this new information.

**Reply by jbadeau** [reply]
Great tool by the way

**Reply by jbadeau** [reply]
Do you need an issue for this?

**Reply by aalmiray** [reply]
That’d be great, yes thank you.
